cmd/config-seed/res/properties/edgex-support-rulesengine;docker/application.properties

```diff
@@ -26,24 +26,6 @@ heart.beat.msg=Support Rules Engine data heart beat
 app.open.msg=This is the Support Rules Engine Microservice.
 # set port (override Spring boot default port 8080 )
 server.port=48075
-#-----------------Export Service Config----------------------------------------
```

Comment:
This section need to be retained, reworked and renamed to `App Service Configurable Rues config`. See below for required elements.